.eslintrc.js

```diff
@@ -370,41 +371,29 @@ module.exports = {
       files: ['x-pack/plugins/canvas/**/*'],
       plugins: ['prettier'],
       rules: {
-        // preferences
-        'comma-dangle': [2, 'always-multiline'],
```

Comment:
Why'd you remove the comma dangle and spacing rules? Perhaps, they're upstream rules now and so they're redundant here?